### PR TITLE
docs: re enable devnet infra guide in gitbook

### LINKED ISSUE #630: docs: re-enable devnet infra guide in gitbook

### LINKED ISSUE #653: docs: show link to github repo
Currently the docs don't include a github repo link anywhere

### PR DESCRIPTION
**Pull request by uri-99**
# Changes

* closes #653 
* add infra guide with steps for devnet, explorer, metrics, etc.

### CODE REVIEW

File: docs/guides/5_setup_aligned.md [hunk 1]
```diff
@@ -65,102 +54,107 @@ And you can use this file as an example on how to fill it:
 
 After having the env setup, run in different terminals the following commands to boot Aligned locally:
 
+## Anvil
+
+To start anvil, a local Ethereum devnet, for you to deploy and interact with the contracts:
```
Comment (entropidelic) [inline docs/guides/5_setup_aligned.md]:
```suggestion
To start anvil, a local Ethereum devnet with all necessary contracts already deployed and ready to be interacted with, run:
```

File: docs/SUMMARY.md [hunk 2]
```diff
@@ -23,13 +23,12 @@
...
-* [Contract Addresses](guides/4_contract_addresses.md)
-* [Generating & submitting proofs of Rust code with ZKRust](guides/0_5_using_zkrust.md)
-
-<!-- * [Setup Aligned](developer_guides/2_setup_aligned.md) -->
+* [Generating & submitting proofs of Rust code with ZKRust](guides/1_using_zkrust.md)
+* [SDK](guides/2_SDK.md)
+* [Integrating Aligned into your Application](guides/3_integrating_aligned_into_your_application.md)
+* [Generating proofs for Aligned](guides/4_generating_proofs.md)
+* [Setup Aligned Infrastructure Locally](guides/5_setup_aligned.md)
+* [Contract Addresses](guides/6_contract_addresses.md)
```
Comment (entropidelic) [inline docs/SUMMARY.md]:
```suggestion
* [SDK](guides/2_SDK.md)
* [Integrating Aligned into your Application](guides/3_integrating_aligned_into_your_application.md)
* [Generating proofs for Aligned](guides/4_generating_proofs.md)
* [Generating & submitting proofs of Rust code with ZKRust](guides/1_using_zkrust.md)
* [Setup Aligned Infrastructure Locally](guides/5_setup_aligned.md)
* [Contract Addresses](guides/6_contract_addresses.md)
```

File: docs/guides/5_setup_aligned.md [hunk 3]
```diff
@@ -220,15 +214,15 @@ make operator_mint_mock_tokens CONFIG_FILE=<path_to_config_file>
 make operator_deposit_into_mock_strategy CONFIG_FILE=<path_to_config_file>
 ```
 
-#### Deposit Strategy tokens in Holesky/Mainnet
+### Deposit Strategy tokens in Holesky/Mainnet
 
 EigenLayer strategies are available in [eigenlayer-strategies](https://holesky.eigenlayer.xyz/restake).
 
 For Holesky, we are using [WETH](https://holesky.eigenlayer.xyz/restake/WETH) as the strategy token.
```
Comment (NicolasRampoldi) [inline docs/guides/5_setup_aligned.md:230]: Should we add that we now support all tokens?

File: docs/guides/5_setup_aligned.md [hunk 4]
```diff
@@ -220,15 +214,15 @@ make operator_mint_mock_tokens CONFIG_FILE=<path_to_config_file>
...
 
 To obtain HolETH and swap it for different strategies, you can use the following [guide](https://docs.eigenlayer.xyz/eigenlayer/restaking-guides/restaking-user-guide/testnet/obtaining-testnet-eth-and-liquid-staking-tokens-lsts).
 
-#### Config
+### Config
 
 There is a default configuration for devnet purposes in `config-files/config.yaml`.
 Also, there are 3 different configurations for the operator in `config-files/devnet/operator-1.yaml`, `config-files/devnet/operator-2.yaml` and `config-files/devnet/operator-3.yaml`.
```
Comment (NicolasRampoldi) [inline docs/guides/5_setup_aligned.md]: I think the config file used as an example here is old, it's missing:
```
enable_metrics
metrics_ip_port_address
max_batch_size
```
 in the operator config

File: docs/guides/5_setup_aligned.md [hunk 5]
```diff
@@ -322,205 +335,169 @@ ecdsa:
   private_key_store_password: <ecdsa_private_key_store_password>
 ```
```
Comment (NicolasRampoldi) [inline docs/guides/5_setup_aligned.md:357]: Batcher config seems to be old, it's missing:
```
max_proof_size
max_batch_size
eth_ws_reconnects
pre_verification_is_enabled
 ```

File: docs/guides/5_setup_aligned.md (hunk 3 above)
Comment (NicolasRampoldi) [reply docs/guides/5_setup_aligned.md:230]: Nvm, let's leave it for another PR.

File: docs/guides/5_setup_aligned.md (hunk 4 above)
Comment (NicolasRampoldi) [reply docs/guides/5_setup_aligned.md]: Solved in [44a26ab](https://github.com/yetanotherco/aligned_layer/pull/631/commits/44a26abfc3b2e462c51f305397b5532d3ee122e2)

File: docs/guides/5_setup_aligned.md (hunk 5 above)
Comment (NicolasRampoldi) [reply docs/guides/5_setup_aligned.md:357]: Solved in [44a26ab](https://github.com/yetanotherco/aligned_layer/pull/631/commits/44a26abfc3b2e462c51f305397b5532d3ee122e2)

**Review by glpecile — APPROVED** [review]

File: docs/guides/setup_holesky.md
Comment (glpecile) [inline docs/guides/setup_holesky.md:1]: Where is this file used?

Comment (uri-99) [reply docs/guides/setup_holesky.md:1]: This guide is to deploy to holesky, as users don't need this guide, it is not included in the gitbook, but it is still present on the repo

**Review by NicolasRampoldi — APPROVED** [review]

**Pull request merged**